scoop/Dockerfile.insider

```diff
@@ -0,0 +1,12 @@
+FROM mcr.microsoft.com/windowsservercore-insider:10.0.17744.1001
+
+ENV SCOOP "C:\scoop"
+ENV SCOOP_HOME "C:\scoop\apps\scoop\current"
+
+RUN powershell.exe -NoLogo -Command " \
+        iwr -UseBasicParsing 'https://get.scoop.sh' | iex; \
+        scoop install 7zip git sudo innounp lessmsi dark; \
```

Comment:
Same here, better without `scoop install` any other tools.

Reply:
Understood and agreed. But FYI: 7zip, sudo, innounp, lessmsi, and dark are pre-installed as are sometimes used by scoop to install other apps. They can be jettisoned as they are certainly optional. In fact, 7zip is auto-installed when first needed. But git is required by scoop to update itself, or update other buckets, so it probably needs to be installed. Maybe in a separate layer?